pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -243,10 +243,11 @@
         },
 
         "ubuntu": {
-            "#import": [ "linux" ]
+            "#import": [ "debian" ]
         },
+
         "ubuntu-x64": {
-            "#import": [ "ubuntu", "linux-x64" ]
+            "#import": [ "ubuntu", "debian-x64" ]
         },
 
         "ubuntu.14.04": {
```

Comment:
Given that Jessie (Debian 8.0) was released 4/16/15, it feels a little scary to say that Ubuntu 14.04 (released a year earlier) is a decedent.

RHEL and CentOS are thightly coupled (CentOS takes the RHEL source and basically rebuilds it after removing some RedHat trademarks).

I understand that Ubuntu and Debian started from a shared based, but it feels bad to expose a relationship that could not exist in our runtime graph.

I'm not pushing back on the diff here, that seems reasonable, though I do question how you accurately use these version-less RIDs in any sensible way (e.g. if a newer "debian-x64" updates to some glibc version and toasts your precompiled so for libuv, what do you do?)